src/apps/vpn/ui/screens/home/ViewServers.qml

```diff
@@ -84,16 +84,20 @@ Item {
         }
 
         stackContent: [
-            ServerList {
-                id: singleHopServerList
-                currentServer: {
-                    "countryCode": VPNCurrentServer.exitCountryCode,
-                    "localizedCityName": VPNCurrentServer.localizedExitCityName,
-                    "cityName": VPNCurrentServer.exitCityName,
-                    "whichHop": "singleHopServer"
+            Loader {
+                active: segmentedNav.selectedIndex == 0
```

Comment:
Don't love the comparison using index since it requires us to keep the order of the segmented nav button and view in sync, but would require a refactor/usage of enums to fix. It's also unlikely that we modify this ordering in the future, so I think it's fine here.

Reply:
Removed that, we now check the tabID :)